templates/VPC/vpc-20-private.yaml

```diff
@@ -285,6 +285,12 @@ Outputs:
     Export:
       Name:
         !Join ['-', [!Ref 'AWS::Region', !Sub '${AWS::StackName}', 'VpcCidr']]
+  PublicSubnet:
+    Description: "SubnetId of the public subnet"
+    Value: !Ref PrivateSubnet
```

Comment:
Should this not be '!Ref PublicSubnet' ?

Reply:
Oops, yes -- fixed in 8e3ef04